app/javascript/react/components/BreweryIndexContainer.js

```diff
@@ -30,6 +30,7 @@ const BreweryIndexContainer = props => {
         name = {brewery.name}
         city = {brewery.city}
         state = {brewery.state}
+        image = {brewery.image}
       />
```

Comment:
I know its nitpicky, but when passing props convention is to remove spaces around the `=`
e.g. `state={brewery.state}`